test/gtest/ucp/ucp_test.h

```diff
@@ -221,5 +220,8 @@ std::ostream& operator<<(std::ostream& os, const ucp_test_param& test_param);
     UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, self,   "self") \
     UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, tcp,    "tcp")
 
+#define UCP_INSTANTIATE_ALL_TEST_CASE(_test_case) \
```

Comment:
maybe add this to test_ucp_sockaddr.cc instead?

Reply:
then maybe we need to run all tests with new TL?

Reply:
not sure becuase the other tests run shared memory with ib (shm_ib) which works for the other tests but doesn't really cover the mm case for the client-server flow due to a large worker address - which will be addressed in the future. so in this case i need to make the address smaller so that the test would work with rma and mm - with xpmem.
```
alinas@hpc-test-node ~/workspace/tmp/ucx
$UCX_SHM_DEVICES=xpmem ./test/gtest/gtest --gtest_filter=mm_rdmacm/test_ucp_sockaddr_with_rma_atomic.wireup_for_rma_atomic/0
[     INFO ] checking dc: yes
[     INFO ] checking dcx: yes
[     INFO ] checking ud: yes
[     INFO ] checking udx: yes
[     INFO ] checking rc: yes
[     INFO ] checking rcx: yes
[     INFO ] checking rcx_cm: yes
[     INFO ] checking shm_ib: yes
[     INFO ] checking ugni: no
[     INFO ] checking self: yes
[     INFO ] checking tcp: yes
[     INFO ] checking all_rcdc: yes
[     INFO ] checking all: yes
[     INFO ] checking rc_x: yes
[     INFO ] checking ud_mlx5: yes
[     INFO ] checking shm: yes
[     INFO ] checking mm_rdmacm: yes
[     INFO ] Using random seed of 3258
Note: Google Test filter = mm_rdmacm/test_ucp_sockaddr_with_rma_atomic.wireup_for_rma_atomic/0
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from mm_rdmacm/test_ucp_sockaddr_with_rma_atomic
[ RUN      ] mm_rdmacm/test_ucp_sockaddr_with_rma_atomic.wireup_for_rma_atomic/0
[     INFO ] Testing 1.1.1.5:47408
[       OK ] mm_rdmacm/test_ucp_sockaddr_with_rma_atomic.wireup_for_rma_atomic/0 (106 ms)
[----------] 1 test from mm_rdmacm/test_ucp_sockaddr_with_rma_atomic (106 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test case ran. (107 ms total)
[  PASSED  ] 1 test.
``` 
other tests don't have this limitation so shared memory is tested in them anyway.